pkg/apis/admission/register.go

```diff
@@ -14,39 +14,53 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Code generated by main. DO NOT EDIT.
+
 package admission
 
 import (
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// GroupName is the group name use in this package
+// GroupName specifies the group name used to register the objects.
 const GroupName = "admission.k8s.io"
 
-// SchemeGroupVersion is group version used to register these objects
-var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
+// GroupVersion specifies the group and the version used to register the objects.
+var GroupVersion = v1.GroupVersion{Group: GroupName, Version: "admission"}
```

Comment:
`Version` doesn't seem to be good.

Reply:
also this is for internal types. I thought the generator only does external type until now?!